### PR TITLE
Android: Fix controller input causing screen highlight

### PR DESCRIPTION
**Pull request by ghost**
Fixes: https://github.com/jpd002/Play-/issues/1089

Fix an issue on Android where input from a controller would cause the SurfaceView to become highlighted

### CODE REVIEW

**Comment by Sykomaniac** [comment]
@jpd002 not sure if this is how you want to ultimately implement the fix, the issue is that the SurfaceView is becoming highlighted as Android is, by default, focusing on the view when a user uses a controllers D-pad. The option to disable this behaviour is only available since API 26. There might be a better way to fix this so it works for all API level's? But I'm not an Android dev!

**Comment by Zer0xFF** [comment]
why not add that directly to the main xml? (yes im aware this was only added for api 26, but wouldnt it just be ignored on older versions?)

**Comment by Sykomaniac** [comment]
Will give it a try and update :)

**Comment by Sykomaniac** [comment]
@jpd002 @Zer0xFF changed to just add attibute in required view.

Android does indeed i[gnore unsupported XML attrbiutes](https://developer.android.com/training/basics/supporting-devices/platforms.html#version-codes)

File: build_android/src/main/res/layout/emulator.xml [hunk 1]
```diff
@@ -37,7 +38,7 @@
 				android:id="@+id/emulator_profile"
 				android:layout_width="wrap_content" 
 				android:layout_height="wrap_content" 
-				android:layout_gravity="right"
+				android:layout_gravity="end"
```
Comment (jpd002) [inline build_android/src/main/res/layout/emulator.xml:41]: Is this an intentional change? If not, probably best to leave it out.

**Comment by jpd002** [comment]
Thanks for this!

**Comment by Sykomaniac** [comment]
Just tried the latest build and working perfectly! Thank you for merging it!

**Pull request merged**